k8s-training/variables.tf

```diff
@@ -19,11 +19,12 @@ variable "region" {
   type        = string
 }
 
-# K8s cluster
+# K8s cluster -> Keep commented out to use backend default (reccomended)
+#     if you need to choose manually, choose 1.31 or above
```

Comment:
Your comment says to "keep commented out" but it's not commented out, it's set to null. I think you want to explain that setting to null uses the backend default.